js/bittrex.js

```diff
@@ -583,12 +607,14 @@ module.exports = class bittrex extends Exchange {
 
     async request (path, api = 'public', method = 'GET', params = {}, headers = undefined, body = undefined) {
         let response = await this.fetch2 (path, api, method, params, headers, body);
-        if ('success' in response)
-            if (response['success'])
+        if (response['success'])
```

Comment:
This will break on Python (KeyError) if there's no success or no message key in the response.

Reply:
Aha, I think only in JavaScript, and it didn't make sense there. Feel free to fix it.